modules/serving-transport-encryption-ensuring-seamless-ca-rotation.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/serving-transport-encryption.adoc
+:_content-type: PROCEDURE
+[id="serving-transport-encryption-ensuring-seamless-ca-rotation_{context}"]
+= Ensuring seamless CA rotation
+
+Ensuring seamless CA rotation is essential to avoid service downtime, or to deal with an emergency.
+
+.Procedure
+
+. Create a new CA certificate.
+
+. Add the public key of the new CA certificate to the CA trust bundles as described in the "Trust configuration for OpenShift Serverless Serving components and Knative Services" section. Make sure to also keep the public key of the existing CA.
+
+. Ensure that all clients have consumed the latest set of CA trust bundles. {ServerlessProductName} Serving components will automatically reload the changed CA trust bundles.
+
+. If you have custom workload consuming trust bundles as well, make sure to reload or restart them accordingly.
+
+. Update the `knative-serving-ca-issuer` `ClusterIssuer` to reference the secret containing the new CA certificate.
+
+. Either wait for `cert-manager` to renew all your certificates or enforce it to renew all the certificates. See the `cert-manager` documentation for details.
+
+. Once the CA rotation is fully completed, you can remove the public key of the old CA from the trust bundle configmap. Make sure to wait enough to let all components apply the changes.
+
+[id="additional-resources_serving-transport-encryption-ensuring-seamless-ca-rotation"]
+[role="_additional-resources"]
+== Additional resources
```

Comment:
If this applies to the proc above, would use the `.Additional resources` format:
```suggestion
[role="_additional-resources"]
.Additional resources
```